external/source/rdll_template/README.md

```diff
@@ -0,0 +1,27 @@
+# Reflective DLL Project Template
+This is a project template for Microsoft's Visual Studio to aid in the creation
```

Comment:
Is this for a specific version of Visual Studio or just any version in general? Should be clearer on this given ways that newer versions have broken features of older versions before. Also I think the general title is just Microsoft Visual Studio, but minor comment.

Reply:
I only tested it in VS 2019, but there's hardly any code so it should be compatible with older versions. You may just need to retarget the project.

Reply:
Fair. Should be fine as is then.